build.sbt

```diff
@@ -11,9 +11,9 @@ organization := "com.dwijnand"
 
        sbtPlugin           := true
       sbtVersion in Global := "0.13.16" // must be Global, otherwise ^^ won't change anything
-crossSbtVersions           := List("0.13.16", "1.0.0-RC3")
+crossSbtVersions           := List("0.13.16", "1.1.0")
```

Comment:
The rest looks good, except for this change. I'd like to test that this plugin works against versions of sbt prior to 1.1.

Now that doesn't have to be 1.0.0-RC3, so if you want you can update it to 1.0.0.